packages/indexer-client/src/types/serverModelTypes.ts

```diff
@@ -178,6 +178,19 @@ export interface IndexerServerMarketSnapshot {
   oracle_prices: Record<string, string>;
 }
 
+export interface IndexerServerStakingV2PoolSnapshot {
+  timestamp: string;
+  cumulative_staked: string;
+  cumulative_unstaked: string;
+  number_of_stakers: string;
+}
+
+export interface IndexerServerStakingV2Staker {
+  address: string;
+  stake_amount: string;
+  pool_share: string;
```

Comment:
I'm surprised it isn't a `number` that Chang returns here :o

Reply:
It is number actually, my bad.